### ISSUE TITLE
In Menu -> View : Layout and Selection filter have the same mnemonics letter

### ISSUE THREAD

**Issue by JLWALTENER** [musescore/MuseScore#28587]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Have a score open
2. Clic on Alt+V to open the View menu
3. See at the menu : 'Layout' and 'Selection filter' share the same mnemonics letter = L

Actual Behavior : Press Alt+V+L opens the Layout panel, but "Selection filter" is not accessible via this mnemonics.

### Supporting files, videos and screenshots

![Image](https://github.com/user-attachments/assets/06aea9a6-72a0-4b19-a819-1189261f293e)

### What is the latest version of MuseScore Studio where this issue is present?

Latest release 4.5.2 and Nightlies

### Regression

No.

### Operating system

Windows 11 and probably all OS

### Additional context

Suggestion : 
As the Instruments panel have been replaced by the Layout panel.
The mnemonics of Instruments was **U**, Layout could use also **U**.

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by PtJade-Ceramic** [comment]
Problems on mnemonics should be reported in [Discord](https://discord.com/channels/818804595450445834/818852942882275379/1387484819553914920) rather than here.

I had reported this problem as follows:

>> This is something that should not have been reported on GitHub. However, it is fixed now anyway, via fixing a translation on Transifex.
> 
> There are many other similar conflicts even in the latest version (specific for Simplified Chinese (zh CN)), as is illustrated hereunder:
> <img width="194" alt="Image" src="https://github.com/user-attachments/assets/096576d8-a634-472b-ab99-ff1097cbe26b" />
> and can you tell me how to modify mnemonic settings, which I can't find now on [MuseScore | Transifex](https://app.transifex.com/musescore/musescore/translate/#zh_CN/musescore/431867054), please? 

 _Originally posted by @PtJade-Ceramic in [#27323](https://github.com/musescore/MuseScore/issues/27323#issuecomment-2835170983)_

**Comment by JLWALTENER** [comment]
I disagree with you, this is an issue involving developpers team rather than translation.
So it is important that developpers can see the issue more precisely than the discord channel is volatile and not traced.

**Comment by cbjeukendrup** [comment]
That's right, this one has to be fixed in the source code (because it is in "source code English") so should be reported here, while issues with translations have to be fixed on Transifex, so should be reported elsewhere.

**Comment by Jojo-Schmitz** [comment]
Seems "e" is free, so "S&election" should work

**Comment by Jojo-Schmitz** [comment]
For now I've fixed the en_US, en_GB and de translations

**Comment by Jojo-Schmitz** [comment]
See #28327
